lazyflow/operators/ioOperators/opExportMultipageTiff.py

```diff
@@ -81,54 +84,32 @@ def run_export(self) -> None:
         if path.exists():
             path.unlink()
 
-        self._page_buf = _NdBuf(self._opReorderAxes.Output.meta.shape[:-2])
+        dtype = self._opReorderAxes.Output.meta.dtype
+        if isinstance(dtype, type):
```

Comment:
I've seen this occurring in many places. We probably should have an utility func:

```python
def as_dtype(obj):
    """Like :func:`numpy.dtype`, but also works with NumPy scalar types."""
    if isinstance(obj, type):
        if issubclass(obj, numpy.generic):
            return obj().dtype
        raise ValueError(f"{obj} is not a NumPy scalar type")
    return numpy.dtype(obj)
```

But I'm not sure whether it's justified.

Reply:
something like that would be clearly useful